ports/templates/ports/builds_filtered_table.html

```diff
@@ -54,12 +54,12 @@ <h4 class="text-secondary">Showing {% if port_name != '' %}results for <i>"{{ po
     </thead>
     {% for build in builds %}
         <tr>
-            <td><a href="/ports/{{ build.port_name }}">{{ build.port_name }}</a></td>
+            <td><a href="{% url 'port_detail' build.port_name %}">{{ build.port_name }}</a></td>
             <td>{{ build.builder_name.name }}</td>
-            <td><a target="_blank" href="{{ build.build_url }}">{{ build.build_id }}</a></td>
-            <td>{{ build.time_start }} </td>
+            <td><a target="_blank" href="https://build.macports.org/builders/ports-{{ build.builder_name.name }}-builder/builds/{{ build.build_id }}">{{ build.build_id }}</a></td>
```

Comment:
I would turn this into a simple python function and call the function from the template instead.

Reply:
Thank you